source/assets/stylesheets/styles/_project_story_large_tile.scss

```diff
@@ -0,0 +1,95 @@
+.project-story-large-tile {
+  box-sizing: border-box;
+  display: block;
+  margin-bottom: 6px;
+  padding-right: 6px;
+  text-decoration: none;
+
+  @include breakpoint($tablet-breakpoint) {
+    float: left;
+    margin-bottom: 0;
+    width: 66%;
+  }
+}
+
+.project-story-large-tile__container {
+  background-color: $project-story-tile__background-color;
+  box-sizing: border-box;
+  border: 1px solid $project-story-tile__border-color;
+  padding: 6px;
+  width: 100%;
+
+  &:hover {
+    background-color: $project-story-tile__hover-color;
+  }
+}
+
+.project-story-large-tile__image-container {
+  box-sizing: border-box;
+}
```

Comment:
This seems like an unnecessary class and additional `div` to the markup. I think we could get away with just:

```CSS
.project-story-X-tile
  .project-story-X-tile__container
  img.project-story-X-tile__image
  .project-story-X-tile__content
     ...
```